Tasks/Maven/task.json

```diff
@@ -71,6 +71,19 @@
             "required": true,
```

Comment:
Make sure to increment the version number above. If it doesn't change, no one will get the new version of the task.